### DISCUSSION TITLE
Encountering an error while trying to run tests locally with JEST_STATE_SYMBOL

### DISCUSSION THREAD

**Discussion by elevenpassin** [webpack/webpack#16742] [Q&A]
Hello, so I am trying to write some test for a patch I'm submitting, but trying to run tests locally gives me this error below. Is there a way to fix it? I checked the docs but I can't really find any thing mentioning this particular property the test files are looking for.

```sh
 FAIL  test/ConfigCacheTestCases.longtest.js
  ● Test suite failed to run

    Unable to find JEST_STATE_SYMBOL in Symbol(jest-native-promise), Symbol(jest-native-now), Symbol(jest-native-read-file), Symbol(jest-native-write-file), Symbol(jest-native-exists-file), Symbol($$jest-matchers-object), Symbol(graceful-fs.queue)

       5 | );
       6 | if (!STATE_SYM) {
    >  7 |      throw new Error(
         |            ^
       8 |              `Unable to find JEST_STATE_SYMBOL in ${Object.getOwnPropertySymbols(global)
       9 |                      .map(s => s.toString())
      10 |                      .join(", ")}`

      at Object.<anonymous> (test/helpers/createLazyTestEnv.js:7:8)
      at Object.<anonymous> (test/ConfigTestCases.template.js:11:27)
```

**Comment by alexander-akait** [comment]
@buoyantair Sorry for delay, do you still the problem?

@TheLarkInn Did we solve it?

**Reply by elevenpassin** [reply]
Hi @alexander-akait , apologies for a very late reply 😓  Yeah the problem is still around, I'm on Windows 11 and trying to run all tests (as part of setting up the project)

**Reply by alexander-akait** [reply]
Yeah, jsut use old version of node.js, we will fix it soon, there is a bug on the jest side
